config/karma-conf.js

```diff
@@ -67,7 +67,7 @@ module.exports = function(config) {
 		// test results reporter to use
 		// possible values: 'dots', 'progress'
 		// available reporters: https://npmjs.org/browse/keyword/karma-reporter
-		reporters: ['progress', 'coverage', 'threshold'],
+		reporters: ['mocha', 'coverage', 'threshold'],
```

Comment:
To quote my item 2:

Update karma config and use mocha reporter instead of progress, this is good because progress does not show which test fails when you do sandbox.verifyAndRestore in afterEach; though generally one should avoid assert (sandbox.verify is implied assert) in afterEach because it cause mocha to abort all subsequent tests.